src/test/ui/generic-associated-types/bugs/issue-86218.rs

```diff
@@ -1,7 +1,4 @@
-// check-fail
-// known-bug
-
-// This should pass, but seems to run into a TAIT issue.
+// check-pass
```

Comment:
Can you also move this into the parent directory? The intention was for this to be "known-bugs", but I guess I could have named better.